example/classifier/odp_classifier.c

```diff
@@ -783,6 +783,9 @@ int main(int argc, char *argv[])
 	if (odp_pool_destroy(pool))
 		ODPH_ERR("err: odp_pool_destroy error\n");
 
+	if (args->if_name)
+		free(args->if_name);
```

Comment:
A very minor thing and a matter of taste, I guess, but the 'if' is redundant since free(0) is a no-op.